pkg/storage/store.go

```diff
@@ -3228,26 +3234,24 @@ func (s *Store) HandleRaftResponse(ctx context.Context, resp *RaftMessageRespons
 				repl.cancelPendingCommandsLocked()
 			}
 			repl.mu.Unlock()
-			replCtx := repl.AnnotateCtx(ctx)
 			added, err := s.replicaGCQueue.Add(
 				repl, replicaGCPriorityRemoved,
 			)
 			if err != nil {
-				log.Errorf(replCtx, "unable to add to replica GC queue: %s", err)
+				log.Errorf(ctx, "unable to add to replica GC queue: %s", err)
 			} else if added {
-				log.Infof(replCtx, "added to replica GC queue (peer suggestion)")
+				log.Infof(ctx, "added to replica GC queue (peer suggestion)")
 			}
 		case *roachpb.StoreNotFoundError:
 			log.Warningf(ctx, "raft error: node %d claims to not contain store %d for replica %s: %s",
 				resp.FromReplica.NodeID, resp.FromReplica.StoreID, resp.FromReplica, val)
-			return val.GetDetail()
+			return val.GetDetail() // close Raft connection.
```

Comment:
ignorable nit: no punctuation in inline comments 🙈

Reply:
🙈